set.mm

```diff
@@ -47381,6 +47411,16 @@ practical reasons (to avoid having to prove sethood of ` A ` in every use
       BJZACHADCKPQACBOLMN $.
   $}
 
+  ${
+    $d x y A $.
+    $( A condition which implies the existential uniqueness of elementhood.
+       (Contributed by AV, 4-Jan-2022.) $)
+    eqeuel $p |- ( ( A =/= (/) /\ A. x A. y ( ( x e. A /\ y e. A ) -> x = y ) )
+                   -> E! x x e. A ) $=
```

Comment:
"A condition which implies the existence of a unique element of a class." ?

Is there in set.mm a theorem of this kind for any predicate ? Something like
x = y -> ( ph <-> ps ) (with DVs so that ph is P(x) and ps is P(y))
x = z -> ( ph <-> ch ) (with DVs so that ch is P(z))
 ==>
A. y A. z ( ( ps /\ ch ) -> y = z ) <-> E* x ph

This could then be applied to ph being "x e. A" to prove your theorem.

Reply:
The specialty of this theorem is the condition A =/= (/), which is used instead of E. x ph (with x e. A as ph in this case).

Reply:
I saw that. So you have a condition for existence and a condition for uniqueness; you deduce from these two a condition for existential uniqueness (namely, the conjunction of said conditions).

Reply:
Yes, correct. This corresponds to the nearby Theorems of this kind, e.g., ~n0moeu.

Reply:
My point is that a more natural proof of eqeuel would be to deduce it from the theorem I sketched above. So that's a future project.
For now, I proposed above a rewording of the comment.  "elementhood" is synonymous with "membership", so the comment means something different from what is proved.

Reply:
OK, I understood your first remark as question, not as proposal for a rewording. And yes, your suggestion is more appropriate. I changed the wording in my latest commit.